SECURITY-INSIGHTS.yml

```diff
@@ -0,0 +1,38 @@
+header:
+  schema-version: '1.0.0'
+  expiration-date: '2025-01-25T01:00:00.000Z'
+  last-updated: '2024-01-25'
+  last-reviewed: '2024-01-25'
+  project-url: https://github.com/cilium/cilium
+  license: https://github.com/cilium/cilium/blob/main/LICENSE
+project-lifecycle:
+  status: active
+  bug-fixes-only: false
+  core-maintainers:
+    - https://github.com/cilium/cilium/blob/main/MAINTAINERS.md
+  roadmap: https://docs.cilium.io/en/latest/community/roadmap
+contribution-policy:
+  accepts-pull-requests: true
+  accepts-automated-pull-requests: true
+dependencies:
+  third-party-packages: true
+  dependencies-lists:
+    - https://github.com/cilium/cilium/blob/main/go.mod
+distribution-points:
+  - https://github.com/cilium/cilium
```

Comment:
Perhaps this should include the quay.io repos and maybe dockerhub ones too?

Reply:
For sure !